x-pack/plugins/xpack_main/index.js

```diff
@@ -81,7 +82,19 @@ export const xpackMain = (kibana) => {
           }),
           type: 'string', // TODO: Any way of ensuring this is a valid email address?
           value: null
-        }
+        },
+        [CONFIG_ROLLUPS]: {
```

Comment:
if i could just get a second pair of eyes on this and make sure i handled this correctly for i18n... i renamed the strings here from `kbn.advancedSettings.[name]` to `xpack.main.rollups.[name]` since it's no longer living in the default settings, and removed them from the `zh-CN.json` file.

Also, side note, I removed the last sentence in the string `"Refresh the page to apply the changes."` since it looked like the request was made immediately and that wasn't necessary.